wp-seo.php

```diff
@@ -1,5 +1,10 @@
 <?php
-/*
+/**
+ * @package    WPSEO
+ * @subpackage Main
+ */
+
+/**
```

Comment:
So, could you prefix the contents of the block with a `*` then. Github really makes it look ugly otherwise :wink: